docs/csharp/toc.yml

```
@@ -1134,6 +1124,9 @@ items:
     - name: Declaration statements
       displayName: var, ref
       href: language-reference/statements/declarations.md
+    - name: Exception-handling statements
+      displayName: throw, try, catch, finally, when, error, fault, try-catch, try-finally, try-catch-finally
```

The "try-catch", "try-finally", and "try-catch-finally" aren't needed. Those phrases match the "try", "catch" or "finally" keywords:

```suggestion
      displayName: throw, try, catch, finally, when, error, fault
```